Samples/Samples.iOS/Info.plist

```diff
@@ -28,39 +28,21 @@
 	<key>CFBundleIdentifier</key>
 	<string>com.xamarin.essentials</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
-	<key>CFBundleIconFiles</key>
-	<array>
-		<string>Icon-60@2x</string>
-		<string>Icon-60@3x</string>
-		<string>Icon-76</string>
-		<string>Icon-76@2x</string>
-		<string>Default</string>
-		<string>Default@2x</string>
-		<string>Default-568h@2x</string>
-		<string>Default-Portrait</string>
-		<string>Default-Portrait@2x</string>
-		<string>Icon-Small-40</string>
-		<string>Icon-Small-40@2x</string>
-		<string>Icon-Small-40@3x</string>
-		<string>Icon-Small</string>
-		<string>Icon-Small@2x</string>
-		<string>Icon-Small@3x</string>
-	</array>
+	<string>0</string>
```

Comment:
```suggestion
	<string>1</string>
```

Reply:
I don't have a strong opinion on this. "0" is a valid build number for iOS builds, at least (in contrast to Android, where this is not the case). If you're not planning to update the version and build number, it has not much meaning anyway.